README.md

```diff
@@ -26,18 +26,6 @@ php artisan serve --host=localhost
 npm run dev
 ```
 
-#### Notes
-
-Sometimes, while viewing it live, it will display 500 error.
-This error comes from the hosting provider that the app is hosted on.
-Wait for few minutes, it will display the page properly.
```

Comment:
Give a link to this PR in the `Notes` section.
And let the Issue link remain as it is.

Reply:
Resolved